motion_predict/include/motion_predict/motion_predict.hpp

```diff
@@ -67,17 +68,19 @@ namespace cv{
     \param  ay acceleration noise along y-axis (m^2/s^4)
     \param  process_noise_max is the maximum process noise of the system
     \param  confidence_drop_rate rate of drop in confidence with time
+    \note twist.linear vector is expected to be in the map frame and used as the yaw for future prediction.
+        pose.orientation is not meaningfully used at the moment except passing it on.
     */
-
     std::vector<carma_perception_msgs::msg::PredictedState> predictPeriod(const carma_perception_msgs::msg::ExternalObject& obj, const double delta_t, const double period,const double ax,const double ay ,const double process_noise_max,const double confidence_drop_rate);
 
     /*!
     \brief  Mapping is used to map input range to an output range of different bandwidth.
     \param  obj predicted object
     \param  delta_t time predicted into the future (sec)
     \param  confidence_drop_rate rate of drop in confidence with time
+    \note twist.linear vector is expected to be in the map frame and used as the yaw for future prediction.
```

Comment:
This might be confusing since there is no twist or pose args for this function. Could we add a note to the `obj` comment, i.e. "\param  obj external object; expected in map frame" or similar?

Reply:
ditto

Reply:
Looks like this one still needs to be addressed/updated